cmd/nerdctl/container_restart_linux_test.go

```diff
@@ -71,6 +71,33 @@ func TestRestartPIDContainer(t *testing.T) {
 	assert.Equal(t, baseOutput, sharedOutput)
 }
 
+func TestRestartIPCContainer(t *testing.T) {
+	t.Parallel()
+	base := testutil.NewBase(t)
+
+	const shmSize = "32m"
+	baseContainerName := testutil.Identifier(t)
+	base.Cmd("run", "-d", "--shm-size", shmSize, "--ipc", "shareable", "--name", baseContainerName, testutil.AlpineImage, "sleep", "infinity").AssertOK()
+	defer base.Cmd("rm", "-f", baseContainerName).Run()
```

Comment:
HI @minuk-dev 

The `defer base.Cmd("rm,"...` may run before the `base.Cmd("run," "-d" ...` is better. :-)
If the `nerdctl run` fails, some resources may leak.

Reply:
Applied.